README.md

```diff
@@ -32,6 +32,37 @@ The repository has the following permanent branches:
 
  * **apk** This branch contains the apks for the code in the develop branch. The apks are automatically updated when a commit is pushed to `develop` branch.
 
+
+##Run Server Locally To Test Android
```

Comment:
Could this whole section be under the section "## Setting up the project", some lines above?
Also, instead of being ##, put as ### to be a subsection

Reply:
done.